test/tmx_object_test.dart

```diff
@@ -117,16 +117,18 @@ void main() {
         expect(ps[2].y, equals(Point(x: -78, y: 19).y));
       });
 
-      test('sets isPolygon to true',
-          () => expect(tiledObject.isPolygon, isTrue));
+      test(
+        'sets isPolygon to true',
+        () => expect(tiledObject.isPolygon, isTrue),
+      );
```

Comment:
@erickzanardo even my tired eyes are not good enough, you can see I manually fixed every file on my previous PR and stuff was left unformatted...